### ISSUE TITLE
Duplicate orders due to inheritance? Not sure.

### ISSUE THREAD

**Issue by frankhommers** [mganss/XmlSchemaClassGenerator#246]
I get this error when deserializing:

`InvalidOperationException: 'Order' values must be unique within the same scope. Value '1' is in use. Please change 'Order' property on 'Distance'.`

I generate the stuff with this:
`xscgen -r -t- http://schemas.opengis.net/wfs/2.0/wfs.xsd --prefix=MyNamespace`

When I deserialize stuff, like this:

```csharp
      string test = @"
<wfs:GetFeature xmlns:wfs=""http://www.opengis.net/wfs/2.0"" version=""2.0.0"" service=""WFS"" xmlns:gmgml=""http://www.intergraph.com/geomedia/gml"" count=""300"" >
  <wfs:Query typeNames=""gmgml:MunicipalSectors"" srsName=""EPSG:4326"">
      <fes:Filter xmlns:fes=""http://www.opengis.net/fes/2.0"" xmlns:gml=""http://www.opengis.net/gml/3.2"" xmlns:gmgml=""http://www.intergraph.com/geomedia/gml"">
       <fes:Intersects>
            <fes:ValueReference>GEOMETRY</fes:ValueReference>
            <Point srsName=""EPSG:4326"" xmlns=""http://www.opengis.net/gml"">
                <pos>24.237886249727683 55.73108188168665</pos>
            </Point>
       </fes:Intersects>
    </fes:Filter>
  </wfs:Query>
</wfs:GetFeature>
        ";

      MemoryStream ms = new MemoryStream(System.Text.Encoding.UTF8.GetBytes(test));
      XmlSerializer serializer2 = new XmlSerializer(typeof(GetFeatureType));
      GetFeatureType getFeature = (GetFeatureType)serializer2.Deserialize(ms);
```

I get the InvalidOperationException.

**Comment by mganss** [comment]
There was a bug with `Order` in the presence of substitution groups. It's fixed in 2.0.495.

**Comment by frankhommers** [comment]
Thanks a lot! It works ;-)
